src/agent/coverage/src/sancov.rs

```diff
@@ -337,9 +337,30 @@ impl SancovInlineAccessScanner {
         decoder.set_ip(va);
 
         let mut inst = Instruction::default();
+
         while decoder.can_decode() {
             decoder.decode_out(&mut inst);
 
+            // If no memory operand, there is no table access.
```

Comment:
In a future cleanup pass, we'll factor out all this detection logic into a standalone function or (configurable) struct.